src/components/Splash.js

```diff
@@ -229,34 +269,20 @@ export default ({
             </ul>
           </div>
 
-          <iframe src="https://guild.host/embeds/guild/torontojs/card"
-            loading="lazy"
-            style={{ width: `450px`, height: `360px`, border: `none`, overflow: `hidden` }}
-          />
+          <div id="guild-card-root" style={GuildCardRootStyle} />
 
         </InnerContainer>
-        <InnerContainer style={{ display: `flex`, justifyContent: `space-evenly` }}>
-          <div>
+        <InnerContainer style={{ display: `flex`, justifyContent: `space-between`, overflow: `hidden` }}>
+          <div className="container-column">
```

Comment:
Update this to the `styled-component`:

```suggestion
          <EventColumn>
```

Reply:
Fixed in 1359ff2